.github/workflows/converter_tests.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
       with:
-        path: 'galaxy root'
+        path: 'galaxy'
```

Comment:
Can you revert the changes to this file ?

Reply:
Sure.